src/components/ebay-notice/index.js

```diff
@@ -16,30 +18,53 @@ const constants = {
 
 const defaults = {
     type: 'page',
-    status: 'confirmation'
+    status: 'priority',
```

Comment:
Why is the default status changing to `priority`? Is it because we're defaulting `dismissible` to false? Was this a conscious decision to have an "error" be the default notice?

In my view, the default notice should always be a `confirmation`, but I could be misunderstanding the component's requirements.

Reply:
I changed it to match the docs. I think that by default "error" would be most widely used hence I made `priority` as default